### PR TITLE
Document FSimGate.__init__'s parameters

### PR DESCRIPTION
**Pull request by Strilanc**

### CODE REVIEW

File: cirq/ops/fsim_gate.py [hunk 1]
```diff
@@ -56,6 +56,13 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: Rotation angle, in radians, for the exchange interaction. Controls the inner 2x2 block of the
+                operation's unitary (the middle 2x2 block will equal `cirq.unitary(cirq.Rx(theta))`).
```
Comment (dabacon) [inline cirq/ops/fsim_gate.py]: What does this add that is not in the definition of the class?

The temrs "middle block" and "bottom 1x1 cell" aren't really clear terms in my opinion.

File: cirq/ops/fsim_gate.py [hunk 2]
```diff
@@ -56,6 +56,13 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: Rotation angle, in radians, for the exchange interaction. Controls the inner 2x2 block of the
```
Comment (dabacon) [inline cirq/ops/fsim_gate.py]: not really true.  This is the term in from of  XX+YY.  The exchange interaction is XX+YY+ZZ

**Comment by mpharrigan** [comment]
Something like: theta is the Rx angle in the |01> |10> subspace; phi is the CPhase angle

I think something like this would be helpful and add to the existing definition

File: cirq/ops/fsim_gate.py (hunk 1 above)
Comment (Strilanc) [reply cirq/ops/fsim_gate.py]: It provides useful pop-up documentation when creating the class in pycharm. It places the definition directly adjacent to the use.

File: cirq/ops/fsim_gate.py (hunk 2 above)
Comment (Strilanc) [reply cirq/ops/fsim_gate.py]: Fixed

File: cirq/ops/fsim_gate.py [hunk 3]
```diff
@@ -56,6 +56,12 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: The strength of the XX+YY interaction, in radians.
```
Comment (dabacon) [inline cirq/ops/fsim_gate.py]: (XX+YY) / 2 ?

Comment (Strilanc) [reply cirq/ops/fsim_gate.py]: Maybe we should list which angles correspond to which operations, e.g. the angle at which 01 becomes 10 (phased) for the first time For now I'll merge this; feel free to improve it further.

**Pull request merged**